### DISCUSSION TITLE
Welcome to React Flow Discussions!

### DISCUSSION THREAD

**Discussion by moklick** [xyflow/xyflow#958] [General]
## 👋 Welcome to React Flow Discussions

We want to use this space to discuss new features and breaking changes. 

If you have questions about the usage of React Flow please use the [official docs](https://reactflow.dev), our [discord channel](https://discord.gg/3WAfFSaVKb) or [StackOverflow](https://stackoverflow.com/).

**Comment by tauren** [comment]
Is something wrong with the discord link? Both the link above and the one on your site say "Invalid Invite" when I click on them.

**Reply by moklick** [reply]
thanks, updated!

**Comment by snanne** [comment]
Hi Team, i am facing an issue on onPaneContextMenu, as this is not registering the first right click of an untouched canvas area. Once, touched the subsequent right clicks are registered as expected. Please provide any workaround or is anyone facing similar issues?
I really appreciate any help you can provide.
